api/clients/codecs/blob.go

```diff
@@ -0,0 +1,79 @@
+package codecs
+
+import (
+	"fmt"
+)
+
+// Blob is data that is dispersed on eigenDA.
+//
+// A Blob is represented under the hood by a coeff polynomial
+type Blob struct {
+	coeffPolynomial *coeffPoly
+	// blobLength must be a power of 2, and should match the blobLength claimed in the BlobCommitment
+	// This is the blob length IN SYMBOLS, not in bytes
+	blobLength uint32
```

Comment:
consider renaming to `blobLengthSymbols`

Reply:
Although is symbol standard here? I feel like our whitepaper is the only place I've ever seen it. Should we just use FE for field element everywhere? BlobLengthFE?

Reply:
I originally used `blobLength` to try to keep consistency with the struct in the commitment, but I agree, it's not ideal that I have to put in comments everywhere that it's in symbols

I don't have a preference between `BlobLengthFE` or `BlobLengthSymbols`. Usage of `Symbols` is pretty pervasive in our codebase, but also I don't think anyone would be confused by the `FE` terminology. Do you have a preference?

Reply:
I would personally prefer we get rid of symbols everywhere. But for consistency and minimal diff let’s just use symbols suffix here for now? We can chat during standup what people think of FE/symbols.

Reply:
[c028c00d](https://github.com/Layr-Labs/eigenda/pull/1120/commits/c028c00d6bd07cf6cf1e05a625285d019b323de8)